public/src/css/style.css

```diff
@@ -17,7 +17,8 @@
 }
 
 body, html {
-  font-family: monospace;
+  /* font-family: monospace; */
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
We need to use a monospace font in order to properly align the text inside dependent premises. We can use one other than the default, but for now I would just keep it at the regular monospace font,

Supposed to look like this
![Screen Shot 2021-06-07 at 8 59 14 PM](https://user-images.githubusercontent.com/28243927/121106262-ad08e400-c7d3-11eb-972f-98306dffd87c.jpg)

Looks like this
![image](https://user-images.githubusercontent.com/28243927/121106287-babe6980-c7d3-11eb-9a05-c78c825f83df.png)

Reply:
sure, I'll find a fix for it later